Auto update failing for taskwarrior

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [taskwarrior](https://github.com/termux/termux-packages/blob/865657dd45207aabc388f5358dd62ed4af41568d/packages/taskwarrior) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="bash">
INFO: Updating taskwarrior [Current version: 2.6.2]
INFO: package being updated to 3.0.0.

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 3650k  100 3650k    0     0  5334k      0 --:--:-- --:--:-- --:--:-- 5334k

You are about to commit these changes:

--------------------
diff --git a/packages/taskwarrior/build.sh b/packages/taskwarrior/build.sh
index 1db6cebf9..216cc12e3 100644
--- a/packages/taskwarrior/build.sh
+++ b/packages/taskwarrior/build.sh
@@ -2,10 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://taskwarrior.org
 TERMUX_PKG_DESCRIPTION="Utility for managing your TODO list"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=2.6.2
-TERMUX_PKG_REVISION=3
+TERMUX_PKG_VERSION="3.0.0"
 TERMUX_PKG_SRCURL=https://github.com/GothenburgBitFactory/taskwarrior/releases/download/v${TERMUX_PKG_VERSION}/task-${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=b1d3a7f000cd0fd60640670064e0e001613c9e1cb2242b9b3a9066c78862cfec
+TERMUX_PKG_SHA256=30f397081044f5dc2e5a0ba51609223011a23281cd9947ea718df98d149fcc83
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="libc++, libgnutls, libuuid, libandroid-glob"
 TERMUX_CMAKE_BUILD="Unix Makefiles"
--------------------

bump(main/taskwarrior): 3.0.0


Not committing to Git!

INFO: Trying to build package.
INFO: Package taskwarrior exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building taskwarrior for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0   184k      0 --:--:-- --:--:-- --:--:--  183k
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0  11602      0 --:--:-- --:--:-- --:--:-- 11732
gpg: Signature made Sun 24 Mar 2024 06:16:16 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0   182k      0 --:--:-- --:--:-- --:--:--  183k
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0  11505      0 --:--:-- --:--:-- --:--:-- 11569
gpg: Signature made Fri 22 Mar 2024 06:18:54 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0   173k      0 --:--:-- --:--:-- --:--:--  175k
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0  11953      0 --:--:-- --:--:-- --:--:-- 12072
gpg: Signature made Sun 24 Mar 2024 12:32:08 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libandroid-glob@0.6-2 if necessary...
Found libandroid-glob in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-glob to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libc++@26b if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2024.03.11 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2024.03.11 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency libgmp@6.3.0 if necessary...
Found libgmp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libgmp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-support@28-3 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.17 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.17 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libunistring@1.1 if necessary...
Found libunistring in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libu/libunistring/libunistring_1.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  508k  100  508k    0     0  5041k      0 --:--:-- --:--:-- --:--:-- 5083k
extracting libunistring to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libidn2@2.3.4 if necessary...
Found libidn2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libi/libidn2/libidn2_2.3.4_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  100k  100  100k    0     0  1221k      0 --:--:-- --:--:-- --:--:-- 1212k
extracting libidn2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnettle@3.9.1 if necessary...
Found libnettle in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libn/libnettle/libnettle_3.9.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  393k  100  393k    0     0  3935k      0 --:--:-- --:--:-- --:--:-- 3973k
extracting libnettle to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency nettle@3.9.1 if necessary...
Found nettle in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/nettle/nettle_3.9.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 18208  100 18208    0     0  31952      0 --:--:-- --:--:-- --:--:-- 32000
extracting nettle to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libevent@2.1.12-2 if necessary...
Found libevent in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libe/libevent/libevent_2.1.12-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  194k  100  194k    0     0  1975k      0 --:--:-- --:--:-- --:--:-- 1966k
extracting libevent to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libexpat@2.6.2 if necessary...
Found libexpat in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libexpat to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libnghttp2@1.60.0 if necessary...
Found libnghttp2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libnghttp2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl@1:3.2.1-1 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.2.1-1 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.4.20231001-3 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.4.20231001-3 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.4.20231001-3 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.4.20231001-3 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.2.1 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gdbm@1.23 if necessary...
Found gdbm in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting gdbm to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-posix-semaphore@0.1-3 if necessary...
Found libandroid-posix-semaphore in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-posix-semaphore to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libbz2@1.0.8-6 if necessary...
Found libbz2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libbz2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency bzip2@1.0.8-6 if necessary...
Found bzip2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting bzip2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcrypt@0.2-5 if necessary...
Found libcrypt in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcrypt to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libffi@3.4.6 if necessary...
Found libffi in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libffi to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency liblzma@5.6.1 if necessary...
Found liblzma in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting liblzma to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xz-utils@5.6.1 if necessary...
Found xz-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xz-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite@3.45.2 if necessary...
Found libsqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sqlite@3.45.2 if necessary...
Found sqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting sqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tcl@8.6.14 if necessary...
Found tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency brotli@1.1.0 if necessary...
Found brotli in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting brotli to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libpng@1.6.43 if necessary...
Found libpng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libpng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency freetype@2.13.2 if necessary...
Found freetype in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting freetype to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ttf-dejavu@2.37-8 if necessary...
Found ttf-dejavu in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ttf-dejavu to /home/builder/.termux-build/_cache-all...
Downloading dependency fontconfig@2.15.0 if necessary...
Found fontconfig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fontconfig-utils@2.15.0 if necessary...
Found fontconfig-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fontconfig-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xorg-util-macros@1.20.0 if necessary...
Found xorg-util-macros in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorg-util-macros to /home/builder/.termux-build/_cache-all...
Downloading dependency xorgproto@2023.2 if necessary...
Found xorgproto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xorgproto to /home/builder/.termux-build/_cache-all...
Downloading dependency libxau@1.0.11 if necessary...
Found libxau in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxau to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxdmcp@1.1.5 if necessary...
Found libxdmcp in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxdmcp to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xcb-proto@1.16.0 if necessary...
Found xcb-proto in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xcb-proto to /home/builder/.termux-build/_cache-all...
Downloading dependency python-xcbgen@1.16.0 if necessary...
Found python-xcbgen in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-xcbgen to /home/builder/.termux-build/_cache-all...
Downloading dependency libxcb@1.16.1 if necessary...
Found libxcb in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxcb to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency xtrans@1.5.0 if necessary...
Found xtrans in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting xtrans to /home/builder/.termux-build/_cache-all...
Downloading dependency libx11@1.8.7 if necessary...
Found libx11 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libx11 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxrender@0.9.11 if necessary...
Found libxrender in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxrender to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxft@2.3.8 if necessary...
Found libxft in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxft to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxext@1.3.6 if necessary...
Found libxext in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxext to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libxss@1.2.4 if necessary...
Found libxss in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libxss to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tk@8.6.14 if necessary...
Found tk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python@3.11.8 if necessary...
Found python in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-tkinter@3.11.8 if necessary...
Found python-tkinter in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-tkinter to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency python-ensurepip-wheels@3.11.8 if necessary...
Found python-ensurepip-wheels in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting python-ensurepip-wheels to /home/builder/.termux-build/_cache-all...
Downloading dependency resolv-conf@1.3 if necessary...
Found resolv-conf in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting resolv-conf to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2@10.43 if necessary...
Found pcre2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pcre2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2grep@10.43 if necessary...
Found pcre2grep in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting pcre2grep to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency swig@4.2.1 if necessary...
Found swig in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/s/swig/swig_4.2.1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  6  881k    6 60888    0     0   105k      0  0:00:08 --:--:--  0:00:08  104k
100  881k  100  881k    0     0  1449k      0 --:--:-- --:--:-- --:--:-- 1449k
extracting swig to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libunbound@1.19.3 if necessary...
Found libunbound in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libu/libunbound/libunbound_1.19.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  342k  100  342k    0     0  2977k      0 --:--:-- --:--:-- --:--:-- 3001k
extracting libunbound to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency unbound@1.19.3 if necessary...
Found unbound in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/u/unbound/unbound_1.19.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  555k  100  555k    0     0  4785k      0 --:--:-- --:--:-- --:--:-- 4790k
extracting unbound to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pyunbound@1.19.3 if necessary...
Found pyunbound in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/pyunbound/pyunbound_1.19.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 37868  100 37868    0     0  70591      0 --:--:-- --:--:-- --:--:-- 70517
extracting pyunbound to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgnutls@3.8.3 if necessary...
Found libgnutls in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libg/libgnutls/libgnutls_3.8.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  705k  100  705k    0     0  6520k      0 --:--:-- --:--:-- --:--:-- 6528k
extracting libgnutls to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency gnutls@3.8.3 if necessary...
Found gnutls in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/g/gnutls/gnutls_3.8.3_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 33  171k   33 59517    0     0   106k      0  0:00:01 --:--:--  0:00:01  106k
100  171k  100  171k    0     0   303k      0 --:--:-- --:--:-- --:--:--  303k
extracting gnutls to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libblkid@2.39.3-4 if necessary...
Found libblkid in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libblkid to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libmount@2.39.3-4 if necessary...
Found libmount in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libmount to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsmartcols@2.39.3-4 if necessary...
Found libsmartcols in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsmartcols to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libuuid@2.39.3-4 if necessary...
Found libuuid in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libuuid to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libfdisk@2.39.3-4 if necessary...
Found libfdisk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libfdisk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libcap-ng@2:0.8.4 if necessary...
Found libcap-ng in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libcap-ng to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency util-linux@2.39.3-4 if necessary...
Found util-linux in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting util-linux to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency uuid-utils@2.39.3-4 if necessary...
Found uuid-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting uuid-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency mount-utils@2.39.3-4 if necessary...
Found mount-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting mount-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency fdisk@2.39.3-4 if necessary...
Found fdisk in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting fdisk to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency blk-utils@2.39.3-4 if necessary...
Found blk-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting blk-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/GothenburgBitFactory/taskwarrior/releases/download/v3.0.0/task-3.0.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 3650k  100 3650k    0     0  5845k      0 --:--:-- --:--:-- --:--:-- 5845k
Downloading https://github.com/Kitware/CMake/releases/download/v3.28.3/cmake-3.28.3-linux-x86_64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

 48 50.1M   48 24.5M    0     0  22.8M      0  0:00:02  0:00:01  0:00:01 22.8M
100 50.1M  100 50.1M    0     0  42.0M      0  0:00:01  0:00:01 --:--:--  210M
[termux_setup_cmake]: Applying cmake-c-compiler.patch
-- Android: Targeting API '24' with architecture 'arm64', ABI 'arm64-v8a', and processor 'aarch64'
-- The C compiler identification is Clang 17.0.2
-- The CXX compiler identification is Clang 17.0.2
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /home/builder/.termux-build/_cache/android-r26b-api-24-v1/bin/aarch64-linux-android-clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/builder/.termux-build/_cache/android-r26b-api-24-v1/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Configuring C++17
-- System: Android
-- Performing Test _HAS_CXX17
-- Performing Test _HAS_CXX17 - Success
-- Looking for libshared
-- Found libshared
-- Looking for SHA1 references
-- Looking for timegm
-- Looking for timegm - found
-- Looking for get_current_dir_name
-- Looking for get_current_dir_name - not found
-- Looking for wordexp
-- Looking for wordexp - not found
-- Performing Test HAVE_TM_GMTOFF
-- Performing Test HAVE_TM_GMTOFF - Success
-- Performing Test HAVE_ST_BIRTHTIME
-- Performing Test HAVE_ST_BIRTHTIME - Failed
-- Looking for libuuid
-- Looking for uuid_unparse_lower
-- Looking for uuid_unparse_lower - found
-- Found libuuid
-- Configuring cmake.h
CMake Error at /home/builder/.termux-build/taskwarrior/build/_deps/corrosion-src/cmake/FindRust.cmake:23 (message):
  `rustc` not found in PATH or `/home/builder/.cargo/bin`.

  Hint: Check if `rustc` is in PATH or manually specify the location by
  setting `Rust_COMPILER` to the path to `rustc`.
Call Stack (most recent call first):
  /home/builder/.termux-build/taskwarrior/build/_deps/corrosion-src/cmake/FindRust.cmake:228 (_findrust_failed)
  /home/builder/.termux-build/taskwarrior/build/_deps/corrosion-src/cmake/Corrosion.cmake:63 (find_package)
  /home/builder/.termux-build/taskwarrior/build/_deps/corrosion-src/CMakeLists.txt:73 (include)


-- Configuring incomplete, errors occurred!
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-03-25 00:27:01 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/8413166754">8413166754</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

Can't even get that far, I'm getting:
```log
-- Found Rust: /home/builder/.rustup/toolchains/1.77.0-x86_64-unknown-linux-gnu/bin/rustc (found version "1.77.0") 
-- Using Corrosion as a subdirectory
error: failed to load manifest for workspace member `/home/builder/.termux-build/taskwarrior/src/taskchampion/integration-tests`

Caused by:
  failed to read `/home/builder/.termux-build/taskwarrior/src/taskchampion/integration-tests/Cargo.toml`

Caused by:
  No such file or directory (os error 2)
CMake Error at /home/builder/.termux-build/taskwarrior/build/_deps/corrosion-src/cmake/CorrosionGenerator.cmake:24 (execute_process):
  execute_process failed command indexes:

    1: "Child return code: 101"

Call Stack (most recent call first):
  /home/builder/.termux-build/taskwarrior/build/_deps/corrosion-src/cmake/CorrosionGenerator.cmake:217 (_cargo_metadata)
  /home/builder/.termux-build/taskwarrior/build/_deps/corrosion-src/cmake/Corrosion.cmake:1098 (_generator_add_cargo_targets)
  src/tc/CMakeLists.txt:11 (corrosion_import_crate)


-- Configuring incomplete, errors occurred!
```

Yeah I'm now getting an error related to `ninja`
```log
-- Found Rust: /home/builder/.rustup/toolchains/1.77.0-x86_64-unknown-linux-gnu/bin/rustc (found version "1.77.0") 
-- Using Corrosion as a subdirectory
-- Configuring man pages
-- Configuring done (2.0s)
-- Generating done (0.0s)
CMake Error:
  Running

   '/home/builder/.termux-build/_cache/ninja-1.10.2/ninja' '-C' '/home/builder/.termux-build/taskwarrior/build' '-t' 'recompact'

  failed with:

   ninja: error: build.ninja:162: bad $-escape (literal $ must be written as $$)

  



CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_INSTALL_LIBDIR
    CMAKE_USE_SYSTEM_LIBRARIES
    DOXYGEN_EXECUTABLE


CMake Generate step failed.  Build files cannot be regenerated correctly.
```

--- a/packages/taskwarrior/0001-timestamp.patch32
+++ b/packages/taskwarrior/0001-timestamp.patch32
@@ -1,0 +1,11 @@
+--- a/taskchampion/lib/src/atomic.rs
++++ b/taskchampion/lib/src/atomic.rs
+@@ -22,7 +22,7 @@
+ 
+     unsafe fn from_ctype(self) -> Option<DateTime<Utc>> {
+         if self != 0 {
+-            return Some(utc_timestamp(self));
++            return Some(utc_timestamp(self.into()));
+         }
+         None
+     }

--- a/packages/taskwarrior/build.sh
+++ b/packages/taskwarrior/build.sh
@@ -2,16 +2,39 @@ TERMUX_PKG_HOMEPAGE=https://taskwarrior.org
 TERMUX_PKG_DESCRIPTION="Utility for managing your TODO list"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=2.6.2
-TERMUX_PKG_REVISION=3
+TERMUX_PKG_VERSION=3.0.2
 TERMUX_PKG_SRCURL=https://github.com/GothenburgBitFactory/taskwarrior/releases/download/v${TERMUX_PKG_VERSION}/task-${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=b1d3a7f000cd0fd60640670064e0e001613c9e1cb2242b9b3a9066c78862cfec
+TERMUX_PKG_SHA256=633b76637b0c74e4845ffa28249f01a16ed2c84000ece58d4358e72bf88d5f10
 TERMUX_PKG_AUTO_UPDATE=true
-TERMUX_PKG_DEPENDS="libc++, libgnutls, libuuid, libandroid-glob"
+TERMUX_PKG_DEPENDS="libandroid-glob, libc++, libgnutls, libuuid"
 TERMUX_CMAKE_BUILD="Unix Makefiles"
 
 termux_step_pre_configure() {
+	termux_setup_rust
+	: "${CARGO_HOME:=$HOME/.cargo}"
+	export CARGO_HOME
+
+	rm -rf $CARGO_HOME/registry/src/*/linkme-*
+	cargo fetch --target $CARGO_TARGET_NAME
+
+	local d p
+	p="linkme-android.diff"
+	for d in $CARGO_HOME/registry/src/*/linkme-0.3.8; do
+		patch --silent -p1 -d ${d} \
+			< "$TERMUX_PKG_BUILDER_DIR/${p}"
+	done
+
+	p="linkme-impl-android.diff"
+	for d in $CARGO_HOME/registry/src/*/linkme-impl-0.3.8; do
+		patch --silent -p1 -d ${d} \
+			< "$TERMUX_PKG_BUILDER_DIR/${p}"
+	done
+
 	LDFLAGS+=" -landroid-glob"
+
+	if [ "$TERMUX_ARCH" = "arm" ]; then
+		rustup target add thumbv7neon-linux-androideabi
+	fi	
 }
 
 termux_step_post_make_install() {
@@ -19,4 +42,8 @@ termux_step_post_make_install() {
 		$TERMUX_PREFIX/share/bash-completion/completions/task
 	install -Dm600 -t $TERMUX_PREFIX/share/fish/vendor_completions.d \
 		"$TERMUX_PKG_SRCDIR"/scripts/fish/task.fish
+}
+
+termux_step_post_massage() {
+	rm -rf $CARGO_HOME/registry/src/*/linkme-*
 }

--- a/packages/taskwarrior/linkme-android.diff
+++ b/packages/taskwarrior/linkme-android.diff
@@ -1,0 +1,12 @@
+https://github.com/dtolnay/linkme/pull/66/commits/38cb074be7193d87e386a02c0e3b20a01f8ba6aa
+
+--- a/src/distributed_slice.rs
++++ b/src/distributed_slice.rs
+@@ -156,6 +156,7 @@
+ impl<T> DistributedSlice<[T]> {
+     #[doc(hidden)]
+     #[cfg(any(
++        target_os = "android",
+         target_os = "none",
+         target_os = "linux",
+         target_os = "macos",

--- a/packages/taskwarrior/linkme-impl-android.diff
+++ b/packages/taskwarrior/linkme-impl-android.diff
@@ -1,0 +1,86 @@
+https://github.com/dtolnay/linkme/pull/66/commits/38cb074be7193d87e386a02c0e3b20a01f8ba6aa
+
+--- a/src/declaration.rs
++++ b/src/declaration.rs
+@@ -118,6 +118,7 @@
+         #(#attrs)*
+         #vis static #ident: #linkme_path::DistributedSlice<#ty> = {
+             #[cfg(any(
++                target_os = "android",
+                 target_os = "none",
+                 target_os = "linux",
+                 target_os = "macos",
+@@ -127,25 +128,25 @@
+                 target_os = "freebsd",
+             ))]
+             extern "Rust" {
+-                #[cfg_attr(any(target_os = "none", target_os = "linux"), link_name = #linux_section_start)]
++                #[cfg_attr(any(target_os = "android", target_os = "none", target_os = "linux"), link_name = #linux_section_start)]
+                 #[cfg_attr(any(target_os = "macos", target_os = "ios", target_os = "tvos"), link_name = #macho_section_start)]
+                 #[cfg_attr(target_os = "illumos", link_name = #illumos_section_start)]
+                 #[cfg_attr(target_os = "freebsd", link_name = #freebsd_section_start)]
+                 static LINKME_START: <#ty as #linkme_path::__private::Slice>::Element;
+ 
+-                #[cfg_attr(any(target_os = "none", target_os = "linux"), link_name = #linux_section_stop)]
++                #[cfg_attr(any(target_os = "android", target_os = "none", target_os = "linux"), link_name = #linux_section_stop)]
+                 #[cfg_attr(any(target_os = "macos", target_os = "ios", target_os = "tvos"), link_name = #macho_section_stop)]
+                 #[cfg_attr(target_os = "illumos", link_name = #illumos_section_stop)]
+                 #[cfg_attr(target_os = "freebsd", link_name = #freebsd_section_stop)]
+                 static LINKME_STOP: <#ty as #linkme_path::__private::Slice>::Element;
+ 
+-                #[cfg_attr(any(target_os = "none", target_os = "linux"), link_name = #linux_dupcheck_start)]
++                #[cfg_attr(any(target_os = "android", target_os = "none", target_os = "linux"), link_name = #linux_dupcheck_start)]
+                 #[cfg_attr(any(target_os = "macos", target_os = "ios", target_os = "tvos"), link_name = #macho_dupcheck_start)]
+                 #[cfg_attr(target_os = "illumos", link_name = #illumos_dupcheck_start)]
+                 #[cfg_attr(target_os = "freebsd", link_name = #freebsd_dupcheck_start)]
+                 static DUPCHECK_START: #linkme_path::__private::usize;
+ 
+-                #[cfg_attr(any(target_os = "none", target_os = "linux"), link_name = #linux_dupcheck_stop)]
++                #[cfg_attr(any(target_os = "android", target_os = "none", target_os = "linux"), link_name = #linux_dupcheck_stop)]
+                 #[cfg_attr(any(target_os = "macos", target_os = "ios", target_os = "tvos"), link_name = #macho_dupcheck_stop)]
+                 #[cfg_attr(target_os = "illumos", link_name = #illumos_dupcheck_stop)]
+                 #[cfg_attr(target_os = "freebsd", link_name = #freebsd_dupcheck_stop)]
+@@ -169,14 +170,14 @@
+             static DUPCHECK_STOP: () = ();
+ 
+             #used
+-            #[cfg(any(target_os = "none", target_os = "linux", target_os = "illumos", target_os = "freebsd"))]
+-            #[cfg_attr(any(target_os = "none", target_os = "linux"), link_section = #linux_section)]
++            #[cfg(any(target_os = "android", target_os = "none", target_os = "linux", target_os = "illumos", target_os = "freebsd"))]
++            #[cfg_attr(any(target_os = "android", target_os = "none", target_os = "linux"), link_section = #linux_section)]
+             #[cfg_attr(target_os = "illumos", link_section = #illumos_section)]
+             #[cfg_attr(target_os = "freebsd", link_section = #freebsd_section)]
+             static mut LINKME_PLEASE: [<#ty as #linkme_path::__private::Slice>::Element; 0] = [];
+ 
+             #used
+-            #[cfg_attr(any(target_os = "none", target_os = "linux"), link_section = #linux_dupcheck)]
++            #[cfg_attr(any(target_os = "android", target_os = "none", target_os = "linux"), link_section = #linux_dupcheck)]
+             #[cfg_attr(any(target_os = "macos", target_os = "ios", target_os = "tvos"), link_section = #macho_dupcheck)]
+             #[cfg_attr(target_os = "windows", link_section = #windows_dupcheck)]
+             #[cfg_attr(target_os = "illumos", link_section = #illumos_dupcheck)]
+@@ -184,6 +185,7 @@
+             static DUPCHECK: #linkme_path::__private::usize = 1;
+ 
+             #[cfg(not(any(
++                target_os = "android",
+                 target_os = "none",
+                 target_os = "linux",
+                 target_os = "macos",
+@@ -236,7 +238,7 @@
+                 $item:item
+             ) => {
+                 #used
+-                #[cfg_attr(any(target_os = "none", target_os = "linux"), link_section = $linux_section)]
++                #[cfg_attr(any(target_os = "android", target_os = "none", target_os = "linux"), link_section = $linux_section)]
+                 #[cfg_attr(any(target_os = "macos", target_os = "ios", target_os = "tvos"), link_section = $macho_section)]
+                 #[cfg_attr(target_os = "windows", link_section = $windows_section)]
+                 #[cfg_attr(target_os = "illumos", link_section = $illumos_section)]
+@@ -245,7 +247,7 @@
+             };
+             ($item:item) => {
+                 #used
+-                #[cfg_attr(any(target_os = "none", target_os = "linux"), link_section = #linux_section)]
++                #[cfg_attr(any(target_os = "android", target_os = "none", target_os = "linux"), link_section = #linux_section)]
+                 #[cfg_attr(any(target_os = "macos", target_os = "ios", target_os = "tvos"), link_section = #macho_section)]
+                 #[cfg_attr(target_os = "windows", link_section = #windows_section)]
+                 #[cfg_attr(target_os = "illumos", link_section = #illumos_section)]

